libs/vm_modules/include/vm_modules/ml/optimisation/adam_optimiser.hpp

```diff
@@ -19,11 +19,11 @@
 
 #include "ml/graph.hpp"
 #include "ml/optimisation/adam_optimiser.hpp"
-
 #include "vm_modules/math/tensor.hpp"
```

Comment:
what's going on here? seems to be only a change in whitespace?
is this done automatically by apply_style script? if so how can it be that it wasn't done previously?

Reply:
This is a manual change - I removed a blank line.